### PR TITLE
added qq check

### PR DESCRIPTION
**Pull request by ighmaZ**
i have worked on #50 add q.q check and replaced the given code

### CODE REVIEW

File: src/utils/qametsQatan.ts [hunk 1]
```diff
@@ -96,12 +96,13 @@ const wholeWordsRegx = sequenceSnippets(wholeWords);
...
 export const convertsQametsQatan = (word: string): string => {
   const qametsReg = /\u{05B8}/u;
+  const qametsQatReg = /\u{05C7}/u; 
   const hatefQamRef = /\u{05B3}/u;
-
-  // if no qamets, return
-  if (!qametsReg.test(word)) {
-    return word;
-  }
+
```
Comment (charlesLoder) [inline src/utils/qametsQatan.ts:101]: delete whitespace

**Comment by ighmaZ** [comment]
removed

**Review by charlesLoder — COMMENTED** [review]
Thanks for the help!

You can always run `npm format` to ensure all whitespace and indentation is correct

**Review by charlesLoder — APPROVED** [review]
Thanks!

You can always use `npm run format` to ensure all whitespace and indentation is done programmatically

**Pull request merged**